Swift.gitignore

```diff
@@ -38,7 +38,9 @@ playground.xcworkspace
 # Packages/
 # Package.pins
 # Package.resolved
+# *.xcodeproj
```

Comment:
Really no need to give the wrong idea here: xcodeproj files are very needed in almost all the iOS/macOS projects of the world. So I wouldn't add this commented line.

Reply:
This is specifically for swift packages; you open the Package.swift file with Xcode and all your code is brought in through that. The xcodeproj file can generated using `swift package generate-xcodeproj`.

If you're still not convinced though I'll remove the line.

Reply:
Currently, I'm not used enough to Xcode 11 beta SPM to answer if there are better alternatives. I've ping previous committers.